deps/jerry/api.cc

```diff
@@ -1065,20 +1067,21 @@ MaybeLocal<UnboundScript> ScriptCompiler::CompileUnboundScript(
       }
   }
 
+  jerry_value_t file_name_jerry = jerry_create_string_from_utf8((const jerry_char_t*)*file_name);
```

Comment:
There is no need to do these conversions, the jerry value stored in source->resource_name should be passed directly.